### PR TITLE
Redis stability test

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Comment by howardjohn** [comment]
@mandarjog this has the prometheus metrics stuff we were talking about. I think we can reuse most of the code in future tests we add like the mysql one. Right now its published to my docker repo but we can get it set up on an official one soon

File: perf/stability/redis/templates/redis.yaml [hunk 1]
```diff
@@ -0,0 +1,399 @@
+---
+# Source: redis/templates/secret.yaml
```
Comment (mandarjog) [inline perf/stability/redis/templates/redis.yaml]: make dynamic or specify source like `go generate`

**Comment by howardjohn** [comment]
@mandarjog comments are addressed

**Comment by howardjohn** [comment]
@mandarjog can you take another look?

File: perf/stability/redis/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Redis
+
+This test installs an instance of Redis using the [stable/redis](https://github.com/helm/charts/tree/master/stable/redis) Helm chart.
+
```
Comment (mandarjog) [inline perf/stability/redis/README.md]: John, can you update the readme to describe the actual test apart from the setup ?
It is doing a `get_time` call right?

Comment (howardjohn) [reply perf/stability/redis/README.md]: I will update the readme. It is just doing a set and get with the time as the value. This is to ensure we aren't failing to write and reading stale entries or something. This has been pretty stable but if it gets flakey we can change it (we want to test Istio not the time it takes Redis to propagate values)

**Pull request merged**